source/_integrations/tuya.markdown

```diff
@@ -43,10 +44,27 @@ ha_platforms:
   - vacuum
 ---
 
-The Tuya integration integrates all Powered by Tuya devices you have added to the Tuya Smart and Tuya Smart Life apps. Tuya officially maintains this integration.
+The Tuya integration allows integrating all Powered by Tuya devices you have added to the Tuya Smart and Tuya Smart Life apps. This integration is officially maintained by Tuya.
 
 All Home Assistant platforms are supported by the Tuya integration, except the lock and remote platform.
 
+## Supported platforms
+
+- **Climate**: air conditioner, heater, thermostat.
+- **Fan**: fan, air purifier, humidifier, dehumidifier.
+- **Light**: light, light strip, ambient light, light string, humidifier's light, ceiling light, ceiling fan light, ambient light, motion sensor light, dimmer switch, dimmer, solar light, remote control.
+- **Scene**: smart scene.
+- **Switch**: switch, socket, power strip, smart kettle, breaker, pet water feeder, air purifier, diffuser, circuit breaker, garage door opener, smart camera, siren alarm, light, ceiling light, solar light, smart electricity meter.
+- **Sensor**: temperature and humidity sensor, door window sensor, smoke detector, gas detector, pir detector, water detector, pm2.5 sensor, emergency button, luminance sensor, vibration sensor, co2 detector, co detector, formaldehyde detector, methane detector, volatile organic compound sensor, pressure sensor, solar light, smart electricity meter.
+- **Binary Sensor**: garage door opener, door window sensor, smoke detector, gas detector, pir detector, water detector, human presence sensor, emergency button, vibration sensor.
+- **Number**: human presence sensor, siren alarm, vibration sensor, dimmer switch, dimmer, coffee maker.
+- **Vacuum**: robot vacuum.
+- **Humidifier**: dehumidifier.
+- **Cover**: curtain, curtain switch, curtain robot.
+- **Camera**: smart camera.
+- **Siren**: smart camera, siren alarm.
+- **Select**: dimmer switch, coffee maker.
```

Comment:
I've removed this. It was replaced by line 49.

Reply:
OK, just to be more detailed.

Reply:
I've removed it, because the details are wrong, vague and really too long.

For example, it says it supports the dimmer switch. This is true, if your dimmer switch has a device category that is supported. For example, tqd isn't supported. But this list doesn't tell that.

I'm not adding this back.

Reply:
Got it, thanks! Removed it.